app/controllers/blazer/queries_controller.rb

```diff
@@ -53,7 +53,11 @@ def show
       data_source = Blazer.data_sources[@query.data_source]
       @bind_vars.each do |var|
         query = data_source.smart_variables[var]
-        if query
+        if query.is_a? Hash
+          @smart_vars[var] = query.map {|k,v| [k,v]}
```

Comment:
Please preserve the existing style where possible - `query.map { |k,v| [k, v] }`.